tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -358,6 +179,243 @@ fn check_achainable_label(
 	AchainableClient::parse(resp)
 }
 
+pub trait AchainableTotalTransactions {
+	// Currently, supported networks: ["Litentry", "Litmus", "Polkadot", "Kusama", "Ethereum", "Khala"]
+	fn total_transactions(
+		&mut self,
+		network: &Web3Network,
+		addresses: &[String],
+	) -> Result<u64, Error>;
+}
+
+const PATH_LENS: usize = 7;
+const A4_ERC20_LIT_ETHEREUM_PATHS: [&str; PATH_LENS] = [
+	"/v1/run/label/b65b955c-63eb-4cdf-acd9-46863f9362f2",
+	"/v1/run/label/02b46446-e0ce-43ac-83f1-7e55a2f590dd",
+	"/v1/run/label/3075da9e-a426-4fe6-bd49-2e0624374326",
+	"/v1/run/label/00b52b3f-da38-4ef1-aeb3-3b5fdb508cfb",
+	"/v1/run/label/61105e67-a432-454b-b7e0-1b67d4a37ac9",
+	"/v1/run/label/79237456-bc9a-4a70-a235-09c0e1d138d2",
+	"/v1/run/label/c9149a7e-ef69-4ae2-83e6-dbf6ebe0f796",
+];
+const A4_LIT_LITENTRY_PATHS: [&str; PATH_LENS] = [
+	"/v1/run/label/2e7e4efb-f64f-4c05-8535-efa14915a566",
+	"/v1/run/label/3268a2f3-b6a5-4055-a7ba-0de414e47b73",
+	"/v1/run/label/7c7dcc3b-7cea-4180-841f-fb4c920afb69",
+	"/v1/run/label/1e40a32d-5da7-4969-9648-b391eab33da7",
+	"/v1/run/label/65796b73-92fd-456e-aa28-75862c1c0cb0",
+	"/v1/run/label/7365f0a2-b69f-465d-b48f-5fe4495bfcaf",
+	"/v1/run/label/dd1bddeb-723a-48e6-b9f0-174b67bd0ff5",
+];
+const A4_LIT_LITMUS_PATHS: [&str; PATH_LENS] = [
+	"/v1/run/label/a3f4d87f-d10e-4e0c-9d1a-e05f7e89ea6b",
+	"/v1/run/label/45c636e1-c34f-4d91-aa84-186ca0ebb3aa",
+	"/v1/run/label/ad95aceb-603d-41c4-997d-df196d9b1f94",
+	"/v1/run/label/10b1725a-eafb-4ee1-bace-ed754e98d309",
+	"/v1/run/label/33ccc2bd-ae38-4e41-82d3-fe522880443b",
+	"/v1/run/label/b8bccd1a-ab90-48c3-bc8b-aca3c0d011a3",
+	"/v1/run/label/38ad2b09-4851-44c7-add5-619499788db0",
+];
+pub trait AchainableA4Holder {
+	// Currently, supported networks: ["Litentry", "Litmus", "Ethereum"]
+	fn lit_holder_on_network(
+		&mut self,
+		network: &Web3Network,
+		address: &str,
+		index: usize,
+	) -> Result<bool, Error>;
+}
+impl AchainableA4Holder for AchainableClient {
+	// consistently holding at least 10 LIT tokens
+	fn lit_holder_on_network(
+		&mut self,
+		network: &Web3Network,
+		address: &str,
+		index: usize,
+	) -> Result<bool, Error> {
+		if index >= PATH_LENS {
+			return Err(Error::AchainableError("Wrong index".to_string()))
+		}
+
+		if *network != Web3Network::Ethereum
+			|| *network != Web3Network::Litentry
+			|| *network != Web3Network::Litmus
+		{
+			return Err(Error::AchainableError("Unsupported network".to_string()))
+		}
+
+		let path = if *network == Web3Network::Ethereum {
+			A4_ERC20_LIT_ETHEREUM_PATHS[index]
+		} else if *network == Web3Network::Litentry {
+			A4_LIT_LITENTRY_PATHS[index]
+		} else {
+			A4_LIT_LITMUS_PATHS[index]
+		};
+
+		let params = ReqParams::new(path);
+		let body = ParamsAccount::new(address).into();
+		let resp = self.post(params, &body)?;
+
+		Self::parse(resp)
+	}
+}
+
+// consistently holding at least 5 DOT tokens
```

Comment:
Shouldn't it be 10 DOTs?

Reply:
According to [the description here](https://idhub.litentry.io/scores/web3-user-score-demo), so it should be 5 DOTs.